ci_framework/roles/reproducer/templates/play.yml.j2

```diff
@@ -1,11 +1,90 @@
 ---
+{% if use_content_provider is defined and use_content_provider | bool -%}
+- name: Configure CRC for insecure registry
+  hosts: crcs
+  vars:
+    content_provider_registry_ip: {{ cifmw_reproducer_ctl_ip4 }}
+  tasks:
+    - name: Set insecure registry on crc node
+      ansible.builtin.import_tasks: ci/playbooks/tasks/set_crc_insecure_registry.yml
+
+- name: "Content Provider"
+  hosts: localhost
+  gather_facts: true
+  vars:
+    cifmw_rp_registry_ip: {{ cifmw_reproducer_ctl_ip4 }}
+    cifmw_rp_registry_firewall: false
+    job_id: "{{ job_id }}"
+  tasks:
+{% raw %}
+    - name: Load env variables
+      ansible.builtin.include_vars:
+        file: "{{ item }}"
+      loop:
+        - "{{ ansible_user_dir }}/{{ job_id }}-params/zuul-params.yml"
+        - "./scenarios/centos-9/base.yml"
+        - "./scenarios/centos-9/content_provider.yml"
+        - "./scenarios/centos-9/zuul_inventory.yml"
+
+    - name: Install necessary tools
+      ansible.builtin.import_role:
+        name: install_yamls_makes
+        tasks_from: make_download_tools
+
+    - name: Deploy registry
+      ansible.builtin.import_role:
+        name: registry_deploy
+
+    - name: Set var for cifmw_operator_build_operators var
+      when:
+        - zuul is defined
+        - "'project' in zuul"
+        - "'short_name' in zuul.project"
+      ansible.builtin.set_fact:
+        cifmw_operator_build_operators:
+          - name: "openstack-operator"
+            src: "~/src/github.com/{{ cifmw_operator_build_org }}/openstack-operator"
+            image_base: >-
```

Comment:
this seems fragile but i am also not sure what we're trying to construct below with the reject and join

i see image_base via grep at 

```
./ci/playbooks/content_provider/content_provider.yml:28:      # It handles the case of setting image_base for
./ci/playbooks/content_provider/content_provider.yml-29-      # openstack-ansibleee-operator and openstack-operator project
./ci/playbooks/content_provider/content_provider.yml-30-      # for openstack-ansibleee-operator, it will return openstack-ansibleee
./ci/playbooks/content_provider/content_provider.yml-31-      # and for openstack-operator, openstack will be returned
./ci/playbooks/content_provider/content_provider.yml-32-      when:
./ci/playbooks/content_provider/content_provider.yml-33-        - zuul is defined
--
./ci/playbooks/content_provider/content_provider.yml-35-        - "'short_name' in zuul.project"
./ci/playbooks/content_provider/content_provider.yml-36-      ansible.builtin.set_fact:
./ci/playbooks/content_provider/content_provider.yml-37-        cifmw_operator_build_operators:
./ci/playbooks/content_provider/content_provider.yml-38-          - name: "openstack-operator"
./ci/playbooks/content_provider/content_provider.yml-39-            src: "~/src/github.com/{{ cifmw_operator_build_org }}/openstack-operator"
./ci/playbooks/content_provider/content_provider.yml:40:            image_base: >-
./ci/playbooks/content_provider/content_provider.yml-41-              {{ zuul.project.short_name | split('-') | reject('search','operator') | join('-') }}


```

Reply:
so it's mostly a copy'n'paste from that ci playbook